addons/mail/static/src/components/message/message.js

```diff
@@ -610,6 +610,15 @@ class Message extends Component {
         this.message.originThread.open();
     }
 
+    /**
+     * @private
+     */
+    _onClickSendAgain(ev) {
+        ev.preventDefault();
+        this.message.update({ hasSendError: false });
+        this.threadView.thread.sender._processMessagesToBeSent();
+    }
```

Comment:
Too technical to be in component + actually calling a private method which is not ok.

Suggestions:
- have a `sendAgain` method in message model
- from there call sender `insertMessageToBeSent` with the message in question
- let the sender model handle the `hasSendError` flag

Reply:
An error message doesn't leave the message queue, so it doesn't need to be inserted again.